src/server/common/controller/generic-page-controller/index.js

```diff
@@ -27,7 +23,7 @@ import { NotImplementedError } from '../../helpers/not-implemented-error.js'
  */
 
 export default class GenericPageController {
-  metrics = createMetricsLogger()
+  logger = createLogger()
```

Comment:
This is the line that caused the issues last time. 😰 

I believe the issue is that every instance of this will create its own logger (and in the unit testing context, that happens a lot).

Do we have another approach we could use (e.g. [instantiate the logger outside of the class, and spawn children](https://github.com/pinojs/pino/blob/main/docs/child-loggers.md)?) that would avoid creating multiple loggers?

Also, do you see the memory leak warnings when you run your tests with this change in?

Reply:
no memory leak warnings when i run tests no